molecular/src/main/kotlin/com/hartwig/actin/molecular/panel/PanelVariantAnnotator.kt

```diff
@@ -198,6 +202,37 @@ class PanelVariantAnnotator(
         }
     }
 
+    private fun sourceEvent(variant: SequencedVariant, paveResponse: PaveResponse, transvarAnnotation: TransvarVariant): String {
+        val transcripts = paveResponse.transcriptImpacts
```

Comment:
I would rename this variable to either `transcriptImpacts` or just `impacts`. The below code now has a bit too much of "transcript" in it! It is quite impressive code btw :)